src/Commands/RunScan.ts

```diff
@@ -219,4 +227,67 @@ export class RunScan implements CommandModule {
       process.exit(1);
     }
   }
+
+  private async validateParams(
+    args: Arguments,
+    scanManager: Scans
+  ): Promise<void> {
+    const projectId = args.project as string;
+    if (!projectId) {
```

Comment:
there is a bug as not specifying project id and specifying crawler url should work